docs/fees_and_rewards/Dynamic_Issuance.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Dynamic Issuance
+sidebar_position: 3
+description: Dynamic Issuance Formulas
+keywords:
+    - fees
+    - rewards
+    - WIP
+last_update:
+  date: 04/16/2024
+  author: Saeid Yazdinejad
+---
+import Collapsible from '@site/src/components/Collapsible/Collapsible';
+
+# Dynamic Issuance
+As opposed to having a fixed issuance rate, the Subspace Network implements a dynamic issuance rate based on network usage, where, roughly, the inflation rate is inversely proportional to the utilization rate of the blockspace. 
+
+TLDR: The farmer who proposed a block gets some fresh SSC + fees, and voters get some fresh SSC regardless of what the proposer got.
+
+
+[Dynamic Issuance Specification](https://www.notion.so/Dynamic-Issuance-Specification-3bdf63955cb943489347889775d71c24?pvs=21)
```

Comment:
Let's avoid linking to Notion. This current md page will supersede the notion page.